app/react/portainer/gitops/utils.ts

```diff
@@ -39,3 +39,15 @@ export function cleanGitRepoUrl(url: string) {
     .replace(/\/$/, '') // if there's a trailing slash, remove it
     .replace(/\.git$/, ''); // if there's a trailing .git extension, remove it
 }
+
+export function getGitRepoCommitUrl(url: string, hash: string) {
```

Comment:
should be in the component file, not exported. as long as we don't need it in other places